docs/UserGuide.md

```diff
@@ -71,15 +74,15 @@ Format: `help`
 
 Adds a student to the list of students.
 
-Format: `add n/NAME t/TELEGRAM_HANDLE e/EMAIL [tag/TAG]…​`
+Format: `add [n/NAME] [t/TELEGRAM_HANDLE] [e/EMAIL] [tag/TAG...]​`
```

Comment:
I don't think name, telegram handle and email are optional fields.

Reply:
Resolved by removing [] for all fields within add apart from [tag/TAG]